driver/web/driver.go

```diff
@@ -97,3 +103,51 @@ func (d *Driver) New(config transaction.Config) (tx transaction.Transaction, err
 	txn.config = config
 	return txn, nil
 }
+
+// monitor the running chrome process to service transactions and recycle at every interval
+func (d *Driver) monitor() (err error) {
+
+	// start chrome process and set process group id to avoid
+	// leaving zombies upon termination
+	d.cmd = exec.Command(d.config.BinaryPath, d.config.BinaryArgs...)
+	d.cmd.SysProcAttr = &syscall.SysProcAttr{Setpgid: true}
+	if err := d.cmd.Start(); err != nil {
+		return fmt.Errorf("replicant-executor: error starting chrome process: %w", err)
+	}
+
+	log.Info("chrome process created").Int("pid", int64(d.cmd.Process.Pid)).Log()
+
+	// TODO: don't panic and figure a better way to surface the errors below.
```

Comment:
Shouldn't the parent process control the logic of the recycle time and signal the child? Making it a push instead of a poll process.

Reply:
This is within the scope of the driver. There is  no "parent process", only driver and transactions consuming driver resources.